@uportal/esco-content-menu/.env

```diff
@@ -10,4 +10,5 @@ VUE_APP_FAVORITES_URI=/api/v4-3/dlm/layout.json
 VUE_APP_USER_INFO_URI=/api/v5-1/userinfo
 VUE_APP_USER_MOREINFO_URI=/api/v5-0/people/me
 VUE_APP_ORG_INFO_URI=/change-etablissement/rest/v2/structures/structs/
+VUE_APP_ORG_SWITCH_URI=/uPortal/p/switchStruct/
```

Comment:
is this a this a new custom portlet?

Reply:
This is a link to a custom portlet/ API (the same as before), but I externalized the value to a generic/anonymized one.

here is the link to the portlet/API that provide on our side [all organizations information and user switch (could be usefull on multi tenancy)](https://github.com/GIP-RECIA/esco-ChangeEtablissement).

If there is a common interest on I can purpose it on the uPortal-contrib. We need a such API and I think to move it as soffit app (but not in next month).

Reply:
Okay so that is being used to switch the active tenant?
I think uPortal may have a built in way of handling that /cc @drewwills

Reply:
It's like that, but we aren't using tenancy feature as we use a pags (a pag that I pushed and is already in uPortal) that filter apps on current organization and DLM aren't managed by schools and there is only one (per servername). This service permit to update a property into user account on ldap and after logout the user to apply the change on CAS (I have a work to avoid a logout on), else on uPortal the change can be made easily as we already have a custom api that permit to update user attributes in live (we do it for user picture). Such adds [can be found here ](https://github.com/GIP-RECIA/recia-uPortal-adds).
Also this service offer a REST API to obtains all Organization informations (it's loaded only once per day expect if you request a refresh by apy call - needed to reload the organization picture).